core/lib/storeCodeFromRoute.ts

```diff
@@ -0,0 +1,49 @@
+import { RawLocation } from 'vue-router'
+import config from 'config'
+import { LocalizedRoute } from './types'
+
+const getNormalizedPath = (matchedRouteOrUrl) => {
+  const matchingPath = typeof matchedRouteOrUrl === 'object' ? matchedRouteOrUrl.path : matchedRouteOrUrl
```

Comment:
we're not prepared here for null
```suggestion
  const matchingPath = matchedRouteOrUrl && matchedRouteOrUrl.path || matchedRouteOrUrl
```